spec/lib/docker-sync/dependencies/package_managers/none_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+RSpec.describe DockerSync::Dependencies::PackageManager::None do
+  it_behaves_like 'a dependency'
+
+  describe '.available?' do
+    subject { described_class.available? }
+    it { is_expected.to be true }
+  end
+
+  describe '.ensure!' do
+    subject { described_class.ensure! }
+    it { is_expected.to execute_nothing }
+  end
+
+  describe '.install_package(package_name)' do
+    let(:user_confirmed?) { true }
+    let(:package_name)    { 'some-package' }
+
+    before do
+      allow(described_class).to receive(:ensure!)
+      allow_any_instance_of(Thor::Shell::Color).to receive_messages(
+        say_status: nil,
+        yes?: user_confirmed?
+      )
+    end
+
+    subject { described_class.install_package(package_name) }
+
+    it 'asks for user confirmation' do
```

Comment:
it doesn't describe what the class is actually expected to do.. I think for this one, we should have just one expectation

```
describe '.install_package(package_name)' do
  let(:package_name) { 'some-package' }
  
  subject { described_class.install_package(package_name) }
  
  it 'tells user to install any of the supported package managers' do
    is_expected_to_raise_error(described_class::NO_PACKAGE_MANAGER_AVAILABLE)
  end
end
```

also, not a big fan of ignoring errors, so I guess we can remove `IgnoreErrorsHelpers`... wdyt?